src/io/flutter/console/FlutterConsoleFilter.java

```diff
@@ -81,16 +81,28 @@ public VirtualFile fileAtPath(@NotNull String pathPart) {
 
     final VirtualFile[] roots = ModuleRootManager.getInstance(module).getContentRoots();
     for (VirtualFile root : roots) {
-      final String baseDirPath = root.getPath();
-      final String path = baseDirPath + "/" + pathPart;
-      final VirtualFile file = LocalFileSystem.getInstance().findFileByPath(path);
-      if (!pathPart.isEmpty() && file != null && file.exists()) {
-        return file;
+      if (!pathPart.isEmpty()) {
+        final String baseDirPath = root.getPath();
+        final String path = baseDirPath + "/" + pathPart;
+        VirtualFile file = findFile(path);
+        if (file == null) {
+          // check example dir too (remove when `example` is a content root: https://github.com/flutter/flutter-intellij/issues/2519)
```

Comment:
This will be easier to track if it is a TODO.

Reply:
👍 

Fixed.